app/views/articles/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<p>
+  <strong>Title:</strong>
+  <%= @article.title %>
+</p>
+ 
+<p>
+  <strong>Text:</strong>
+  <%= @article.text %>
+</p>
+
+<h2>Comments</h2>
+<%= render @article.comments %>
+
+<h2>Add a comment:</h2>
+<%= render 'comments/form' %>
+
+<br>
```

Comment:
`<br>` shouldn't be used for styling purposes. Use a div with padding or margins